src/main/java/org/folio/edge/sip2/repositories/CirculationRepository.java

```diff
@@ -6,17 +6,13 @@
 import static org.folio.edge.sip2.utils.JsonUtils.getSubChildString;
 
 import io.vertx.core.Future;
+import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
 import java.time.Clock;
 import java.time.OffsetDateTime;
 import java.time.ZoneOffset;
 import java.time.format.DateTimeFormatter;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
+import java.util.*;
```

Comment:
I would avoid a wildcard import and just pull in what you need from util. It's fine to do this while developing but should change before merging.